Implement Get/SetLineSize due to no timeline for a 2.6.4 release that would include this.
Useful for applications like xara-xtreme.

Index: src/gtk/slider.cpp
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/src/gtk/slider.cpp,v
retrieving revision 1.58.2.3
retrieving revision 1.58.2.4
diff -u -B -w -p -r1.58.2.3 -r1.58.2.4
--- src/gtk/slider.cpp	2006/03/10 18:45:31	1.58.2.3
+++ src/gtk/slider.cpp	2006/07/27 13:34:33	1.58.2.4
@@ -335,13 +335,24 @@ int wxSlider::GetThumbLength() const
     return (int)ceil(m_adjust->page_size);
 }
 
-void wxSlider::SetLineSize( int WXUNUSED(lineSize) )
+void wxSlider::SetLineSize( int lineSize )
 {
+    double fline = (double)lineSize;
+
+    if (fabs(fline-m_adjust->step_increment) < 0.2) return;
+
+    m_adjust->step_increment = fline;
+
+    GtkDisableEvents();
+
+    gtk_signal_emit_by_name( GTK_OBJECT(m_adjust), "changed" );
+
+    GtkEnableEvents();
 }
 
 int wxSlider::GetLineSize() const
 {
-    return 0;
+    return (int)ceil(m_adjust->step_increment);
 }
 
 bool wxSlider::IsOwnGtkWindow( GdkWindow *window )
